### ISSUE TITLE
Binary Convolutional layer

### ISSUE THREAD

**Issue by deepsemantic** [mrastegari/XNOR-Net#20]
Hi, thanks for sharing the code,  where can I find the implementation of binary Convolutional layer? I only find "BinActiveZ.lua"

**Comment by iridescent303** [comment]
I can't find the K which is metioned in the paper in the codes , do you find it?

**Comment by Dashhh** [comment]
@iridescent303 according to the paper 

> We found that the scaling factors for the weights (α) is much more effective than the scaling factors for the inputs (β). Removing β reduces the accuracy by a small margin (less than 1% top-1 alexnet).

I think they are simply omitting it.

**Comment by iridescent303** [comment]
@Dashhh yeah, I have found only using scaling factor for the inputs instead of K can also get the similar result.

**Comment by Lanweichao** [comment]
> @Dashhh yeah, I have found only using scaling factor for the inputs instead of K can also get the similar result.

hello, I also think they omit the K. Could you tell me where the scaling factor were used in their codes since I did not find it? Thank you very much.
